test/vcf/assembly_checker_test.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "assembly_checker_test_aux.hpp"
+
+namespace ebi
+{	
+	
+	TEST_CASE("Valid fasta and vcf combination", "[assembly_checker]")
```

Comment:
these are not really unitary tests. extract a function from the main loop to be able to test just one VcfVariant here